store/instance.go

```diff
@@ -108,6 +108,39 @@ func (s *InstanceService) FindInstanceList(ctx context.Context, find *api.Instan
 	return list, nil
 }
 
+func (s *InstanceService) CountInstance(ctx context.Context, find *api.InstanceFind) (int, error) {
+	tx, err := s.db.BeginTx(ctx, nil)
+	if err != nil {
+		return 0, FormatError(err)
+	}
+	defer tx.Rollback()
+
+	where, args := findInstanceQuery(find)
+
+	rows, err := tx.QueryContext(ctx, `
+		SELECT COUNT(*)
+		FROM instance as count
+		WHERE `+where,
+		args...,
+	)
+	if err != nil {
+		return 0, FormatError(err)
+	}
+	defer rows.Close()
+
+	count := 0
+	for rows.Next() {
```

Comment:
for single record, please use if 

https://sourcegraph.com/github.com/bytebase/bytebase/-/blob/store/instance.go?L331